Packs/Absolute/ReleaseNotes/1_0_25.md

```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### New: Absolute Modeling Rule
+
+<~XSIAM> Created XDM mapping for key event types (Available from Cortex XSIAM v2.4).</~XSIAM>
+
+#### Parsing Rules
+
+##### New: Absolute Parsing Rule
+
+<~XSIAM> Created log ingestion for better parsing of values from the **objectProperties** field (Available from Cortex XSIAM v2.4).</~XSIAM>
```

Comment:
Since this is only for XSIAM, do we really need the XSIAM tags?

Reply:
The integration has some feature for XSOAR, and it doesn't have any impact on XSIAM customers.